rvsol/test/RISCV.t.sol

```diff
@@ -628,7 +628,7 @@ contract RISCV_Test is CommonTest {
 
     function test_lrw_succeeds() public {
         bytes32 value = hex"1e0acbdd44d41d85";
-        uint64 addr = 0x233f3d38d3ce666b;
+        uint64 addr = 0x233f3d38d3ce6668;
```

Comment:
These random addresses in tests were not properly aligned for the call to succeed. I modified them to be aligned in 4 bytes.